package.json

```diff
@@ -16,6 +16,7 @@
 		"serve-website": "cd website && npm run serve",
 		"watch": "npx chokidar \"*.ts\" \"taqueria-utils/**/*.ts\" \"taqueria-protocol/**/*.ts\" -c \"npm run build || npx play-sound beep.mp3\" --initial",
 		"build-and-watch-vscode": "npx chokidar \"*.ts\" \"taqueria-vscode-extension/**/*.ts\" \"taqueria-protocol/**/*.ts\" -c \"npm run compile -w taqueria-vscode-extension || npx play-sound beep.mp3\" --initial",
+		"pretest": "mkdir -p ~/TVsCE_e2e/vscode-taq-test-project && npm run compile -w taqueria-vscode-extension",
```

Comment:
Is this command needed since it seems like we are creating the directory for tests when the tests are run?

Reply:
Hey @alexzbusko. Yes, we need to create it before we compile the project. And we clean-up and refresh it every test. But if we don't do it before compiling, it won't pick up as a workspace folder.